Tests/testSetup.js

```diff
@@ -0,0 +1,4 @@
+import { configure } from 'enzyme';
+import Adapter from 'enzyme-adapter-react-16';
+
+configure({ adapter: new Adapter() });
```

Comment:
Missing newline at end of file.
Non-blocking.

Reply:
updated in latest commit